doc/user/load_save.md

```diff
@@ -152,7 +152,7 @@ to the numeric values used to represent those categories.
 With a `data::DatasetInfo` object, categorical data can be loaded:
 
  - `data::Load(filename, matrix, info, fatal=false, transpose=true)`
-   * `filename` is a `std::string` with a path to the file to be loaded.
+   * `filename` is a `std::filesystem::path` with a path to the file to be loaded.
```

Comment:
I think writing "a `std::string` or a `std::filesystem::path`" would be better here (and in other places), since readers are very likely to immediately be familiar with `std::string`, but less likely to be familiar with `std::filesystem::path`.  (And I don't want to force them to go find out that they can just pass a string anyway and it'll be fine.)